app-i18n/fcitx-configtool/fcitx-configtool-5.1.7-r1.ebuild

```diff
@@ -15,7 +15,7 @@ S="${WORKDIR}/${MY_PN}-${PV}"
 LICENSE="GPL-2+"
 SLOT="5"
 KEYWORDS="~amd64"
-IUSE="kcm +config-qt test"
+IUSE="kcm +config-qt test +X"
```

Comment:
Why are you using +X here? X / wayland USE flags are typically enabled systemwide e.g. via profile, there doesn't seem to be a reason to enable it by default for users that don't have it enabled.

Reply:
Yes. I shouldn't have turned on X by default in ebuild. it's fixed.